public/stylesheets/local.css

```diff
@@ -202,12 +206,12 @@ table td > .badge {
     white-space: pre;
     color: #444;
     background: #F0F0F0;
-    
+
     padding-top: 4px;
     padding-bottom: 4px;
     padding-left: 2px;
     padding-right: 2px;
-    
+
```

Comment:
Are these whitespace changes necessary?

Reply:
I think my editor made those changes without me realizing since I don't remember removing any whitespace. Should I fix them?

Reply:
I'm a fan of empty lines not having whitespace, so this change is fine by me!

Reply:
if possible, yes

Reply:
I guess either is fine then 😆